### PR TITLE
first commit for turbo 451225 Genesys Cloud and Twilio

### PR DESCRIPTION
**Pull request by carinali100**
First commit for turbo launch ticket https://turbo.corp.google.com/docs/451225#doc-TurboLaunch

### CODE REVIEW

**Review by nategd-google — APPROVED** [review]

**Review by katherinez22 — APPROVED** [review]

**Review by katherinez22 — APPROVED** [review]
Code review was completed on Git. This is the PR to sync Git on Borg and GitHub

File: genesyscloud/.DS_Store
Comment (deliaqi) [inline genesyscloud/.DS_Store:1]: Should we remove this?

File: twilioflex/agent-assist-functions/.nvmrc
Comment (deliaqi) [inline twilioflex/agent-assist-functions/.nvmrc:1]: Is this file necessary?

**Review by deliaqi — APPROVED** [review]

File: genesyscloud/.DS_Store
Comment (carinali100) [reply genesyscloud/.DS_Store:1]: yeah, my workspace automatically added those. Remove them now

File: twilioflex/agent-assist-functions/.nvmrc
Comment (matt-degraffenreid) [reply twilioflex/agent-assist-functions/.nvmrc:1]: No it's not should be in the ignore

Comment (carinali100) [reply twilioflex/agent-assist-functions/.nvmrc:1]: append: that file is needed to tell the user what version of node to use. If they have nvm installed it will switch to that version automatically or prompt them to install it.

File: genesyscloud/frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# UI Module Application Server
+
+## Introduction
+
+The purpose of this guide is to set up an application server that can be used in conjunction with the proxy server in the same directory to render Agent Assist suggestions in GenesysCloud.
+
+Although it is possible to set up this integration deployment on any hosting platform, these instructions will use [Google's Cloud Run](https://cloud.google.com/run/) or [App Engine](https://cloud.google.com/appengine).
+
+## Configuration
```
Comment (nategd-google) [inline genesyscloud/frontend/README.md]: Either link to the offical documents or reorder all of the content to be similar to the official doc. This is all out of order and confusing.

File: twilioflex/README.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+
+# Agent Assist Twilio Flex Integration
+
+This project is a solution for integrating Agent Assist into a Twilio Flex instance.
+
+## Prerequisites
```
Comment (nategd-google) [inline twilioflex/README.md]: Either link to the offical doc or re-order this to be like the offical doc

File: twilioflex/agent-assist-functions/src/assets/main.private.ejs [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+
+    <% if (debug === 'true') { %>
+      <script>
+        window._uiModuleFlags = { debug: true };
+      </script>
+    <% } %>
+    
+    <!-- WebComponents polyfill. Needed for rendering web components in older browsers. -->
+    <script src="https://www.gstatic.com/external_hosted/document_register_element/document-register-element.js"></script>
+    <script src="https://www.gstatic.com/agent-assist-ui-modules/v1.6/container.js"></script>
```
Comment (nategd-google) [inline twilioflex/agent-assist-functions/src/assets/main.private.ejs:22]: We should test and release to 1.7

File: twilioflex/docs/photos/twilioDashboard.png
Comment (nategd-google) [inline twilioflex/docs/photos/twilioDashboard.png:1]: remove

**Review by nategd-google — CHANGES_REQUESTED** [review]

File: twilioflex/docs/photos/twilioDashboard.png
Comment (carinali100) [reply twilioflex/docs/photos/twilioDashboard.png:1]: Done

File: twilioflex/README.md (hunk 2 above)
Comment (carinali100) [reply twilioflex/README.md]: Done, linked to official doc

File: genesyscloud/frontend/README.md (hunk 1 above)
Comment (carinali100) [reply genesyscloud/frontend/README.md]: Done, linked to official doc

File: twilioflex/agent-assist-functions/src/assets/main.private.ejs (hunk 3 above)
Comment (carinali100) [reply twilioflex/agent-assist-functions/src/assets/main.private.ejs:22]: will patch to 1.7 after testing

**Review by nategd-google — APPROVED** [review]
